config/core/build-configs.yaml

```diff
@@ -154,6 +154,13 @@ fragments:
   kselftest:
     path: "kernel/configs/kselftest.config"
 
+  perf:
```

Comment:
This happens to be actually enabled by default in all configs, so we might need a fragment to disable it.  But for now, please drop this fragment and let's see how much extra load it adds to always build it.